co-log/example/Main.hs

```diff
@@ -0,0 +1,24 @@
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE TypeApplications #-}
+
+module Main where
+
+import Colog (LogAction (..), WithLog, cmap, logMsg, usingLoggerT, withLog)
+
+example :: WithLog env String m => m ()
+example = do
+    logMsg "First message..."
+    logMsg "Second message..."
+
+app :: WithLog env String m => m ()
+app = do
+    logMsg "Starting application..."
+    withLog (cmap ("app:" ++)) example
+
+foo :: (WithLog env String m, WithLog env Int m) => m ()
+foo = do
```

Comment:
Is this used anywhere?

Reply:
It's not used currently. I've added this example just to check whether it builds or not. To actually launch this example I need to create some data type `Env` and implement the following instances:

```haskell
instance HasLog Env String m
instance HasLog Env Int m
```

This is more complicated so I decided to have only example and I hope to run it in future...